tests/integration/test_svgp_equivalence.py

```diff
@@ -245,6 +246,7 @@ def optimization_step():
         optimization_step()
 
 
+@pytest.mark.skipif(Version(tf.__version__) <= Version("2.3"), reason="Do not run with TF-2.2")
```

Comment:
Please double-check:
```suggestion
@pytest.mark.skipif(Version(tf.__version__) < Version("2.3"), reason="Do not run with TF-2.2")
```